subsys/bluetooth/controller/ll_sw/ull_peripheral_iso.c

```diff
@@ -104,7 +104,8 @@ uint8_t ll_cis_accept(uint16_t handle)
 	if (conn) {
 		uint32_t cis_offset_min;
 
-		if (IS_ENABLED(CONFIG_BT_CTLR_JIT_SCHEDULING)) {
+		if (IS_ENABLED(CONFIG_BT_CTLR_JIT_SCHEDULING) ||
+		    !IS_ENABLED(CONFIG_BT_CTLR_CIS_ACCEPT_MIN_OFFSET_STRICT)) {
```

Comment:
#59230 is not a solution for ACL event length > 500 us; for a 1M 27 byte PDU ACL events taking 806 us on air (Spec minimum 328 us for 27 bytes even when being currently on 2M PHY), and placing a CIG at 500 us offset would cause ACL events (reserved for 806 us + overheads + ACL window widening) to overlap. https://github.com/zephyrproject-rtos/zephyr/commit/4b90d01398c2e6ac674c708adca51b1e74f002b0 permitted negotiation such that CIG event would be placed after ACL event length time reservations.

The change in this PR, relaxes the cis_min_offset check (that would have ensure non-overlapping ACL event lengths) to permit overlaps by allowing 500 us cis_min_offset. In practice the 2M 27 bytes ACL event would not exceed ~500 us plus overheads.

Reply:
So what we need is a combination of the configurations CONFIG_BT_CTLR_JIT_SCHEDULING, CONFIG_BT_CTLR_CIS_ACCEPT_MIN_OFFSET_STRICT and CONFIG_BT_CTLR_PERIPHERAL_ISO_EARLY_CIG_START. 

Maybe moving the CONFIG_BT_CTLR_CIS_ACCEPT_MIN_OFFSET_STRICT in #59230 below the
```
    if (IS_ENABLED(CONFIG_BT_CTLR_JIT_SCHEDULING) ||
        !IS_ENABLED(CONFIG_BT_CTLR_CIS_ACCEPT_MIN_OFFSET_STRICT)
```
condition?

Reply:
Yes, something like that. But I will wait with this current PR until after UPF completes and evaluate how cis_offset_min are being used by vendors.